docs/src/architecture/adr/user-accounts.md

```diff
@@ -0,0 +1,120 @@
+<!--
+    Copyright 2023 TII (SSRC) and the Ghaf contributors
```

Comment:
Fill this to confluence first, from there it will get placed correctly into the Ghaf documentation